content/blog/cleaning-up-your-test-data-builders.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Cleaning Up Your Test Data Builders"
+date: 2019-03-16T13:24:49+01:00
+publishdate: 2019-03-16T13:24:49+01:00
+lastmod: 2019-03-17T00:00:00+01:00
+image: "images/blog/cleaning-up-your-test-data-builders.jpg"
+tags: [ "Cleaner Code", "Test Automation" ]
+comments: true
+draft: true
+---
+
```

Comment:
Overall: ik zou graag de Object Mother class (A) naast de testcode zien.

Reply:
Die stelt echt niets voor en leidt denk ik af van het doel om de test code leesbaarder te maken. Dit is de inhoud van de A class

```csharp
    class A
    {
        public static PersonBuilder Person => new PersonBuilder();
        public static PersonBuilder Man => Person.IsMan();
    }
```